src/foam/u2/crunch/wizardflow/CreateWizardletsAgent.js

```diff
@@ -17,7 +17,7 @@ foam.CLASS({
   `,
 
   imports: [
-    'capabilities',
+    'capabilities?',
```

Comment:
When does this happen?

Reply:
`capabilities` is supposed to be exported by LoadCapabiltiesAgent but for WizardFlow it was removed from the sequence.